windows/internal/smoke_test.bat

```diff
@@ -51,7 +51,7 @@ set "PATH=%CD%\Python%PYTHON_VERSION%\Scripts;%CD%\Python;%PATH%"
 pip install -q numpy protobuf "mkl>=2019"
 if errorlevel 1 exit /b 1
 
-if "%TEST_NIGHTLY_PACKAGE%" == "1" (
+if NOT "%PYTORCH_BUILD_VERSION%"=="%PYTORCH_BUILD_VERSION:dev=%" (
```

Comment:
This is a very obscure check, could it be replaced with something like `TEST_WHEELS` or `NOT %TEST_CONDA%`?